cypress/integration/WebInterface/Measure/EditMeasure/EditMeasureNameValidations.spec.ts

```diff
@@ -2,8 +2,22 @@ import {OktaLogin} from "../../../../Shared/OktaLogin"
 import {CreateMeasurePage} from "../../../../Shared/CreateMeasurePage"
 import {EditMeasurePage} from "../../../../Shared/EditMeasurePage"
 import {MeasuresPage} from "../../../../Shared/MeasuresPage"
+let measureName = 'TestMeasure' + Date.now()
+let CqlLibraryName = 'TestLibrary' + Date.now()
+let measureScoring = 'Ratio'
 
 describe('Edit Measure Name Validations', () => {
+    before('Create Measure', () => {
+
+        OktaLogin.Login()
```

Comment:
another spot where we can remove the login and logout

Reply:
unnecessary logins and logouts have been removed